src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -554,10 +457,20 @@ int FastRouteCore::addNet(odb::dbNet* db_net,
   return netID;
 }
 
+void FastRouteCore::init_usage()
+{
+  int i;
+
+  for (i = 0; i < yGrid * (xGrid - 1); i++)
+    h_edges[i].usage = 0;
+  for (i = 0; i < (yGrid - 1) * xGrid; i++)
+    v_edges[i].usage = 0;
```

Comment:
Since i is just a loop variable there is no need to put it at the function scope.  Just do
for (int i = 0....